cmd/swarm-smoke/upload_and_sync.go

```diff
@@ -179,7 +183,7 @@ func trackChunks(testData []byte) error {
 // addrs - a slice with all uploaded chunk refs
 // allHostChunks - host->bit vector, showing what chunks are present on what hosts
 // bzzAddrs - host->bzz address, used when determining the most proximate host for a given chunk
-func checkChunksVsMostProxHosts(addrs []storage.Address, allHostChunks map[string]string, bzzAddrs map[string]string) {
+func checkChunksVsMostProxHosts(addrs []storage.Address, allHostChunks map[string]string, bzzAddrs map[string]string) error {
```

Comment:
@acud looks like this is no longer run for `pushsync` - it appears to be only called behind the flag `syncDelay`, and not behind `pushsyncDelay`.

Reply:
i fixed this @nonsense, please have a look at https://github.com/ethersphere/swarm/pull/1793/files#diff-4545af8f111f99eb44a69ef9b620c46fR311